src/packets.c

```diff
@@ -1054,6 +1054,13 @@ TbBool process_dungeon_control_packet_clicks(long plyr_idx)
             player->thing_under_hand = thing->index;
         if ((pckt->control_flags & PCtr_LBtnRelease) != 0)
         {
+          if (player->thing_under_hand > 0)
+          {
+            if (player->controlled_thing_idx != player->thing_under_hand)
+            {
+                player->influenced_thing_idx = player->thing_under_hand;
```

Comment:
This seems like a change unrelated to the console commands right? Does this change stuff in normal play?

Reply:
Some of my commands use player->controlled_thing_idx, so I made ordering a creature (I believe querying one does this already) set this value.

Reply:
I see it's just for order creature.